now-build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -xe
+
+cd ../../
+export NODE_ENV="production"
```

Comment:
Why is this necessary? According to the docs you can set env variables, as you do, in `now.json` and I see that you are already setting `NODE_ENV` to production.

Reply:
It's not obvious, but this build script runs _after_ `yarn install`. If `NODE_ENV=production` is set during yarn install, it won't install devDependencies. So we need to wait until install is done before setting that variable.

Reply:
I think a comment in this script would go a long way :) Thanks for clearing that up!